buildpack.md

```diff
@@ -684,28 +673,6 @@ The purpose of launch is to modify the running app environment using app-provide
 
 **GIVEN:**
 - An OCI image exported by the lifecycle,
-- A shell, if needed,
-
-First, the lifecycle MUST locate a start command and choose an execution strategy.
-
-To locate a start command, the lifecycle MUST follow the process outlined in the [Platform Interface Specification](platform.md).
-
-To choose an execution strategy,
-
-1. **If** a buildpack-provided process type is chosen as the start command,
-   1. **If** the process type has `direct` set to `false`,
-      1. **If** the process has one or more `args`
-         **Then** the lifecycle MUST invoke a command using the shell, where `command` and each entry in `args` are shell-parsed tokens in the command.
-      2. **If** the process has zero `args`
-         **Then** the lifecycle MUST invoke the value of `command` as a command using the shell.
-
-   2. **If** the process type does have `direct` set to `true`,
-      **Then** the lifecycle MUST invoke the value of `command` using the `execve` syscall with values of `args` provided as arguments.
-
-2. **If** a user-defined process type is chosen as the start command,
```

Comment:
This section (branching on buildpack-provided process type vs. user-defined process type) could probably stay.

Reply:
A couple notes on my thinking here:

1. The buildpack-vs-user process type branching here in particular only covers the execution strategy, which will go away as part of this (i.e., all will behave as `direct = true` so there will be no choice).
2. The other bit here is detailing out `command` and `args`. This is also written out in the [launcher section in the platform spec](https://github.com/buildpacks/spec/blob/main/platform.md#launcher), which seems to me to be the appropriate home for that info. So, I point to that to keep it in one place.

Happy to hear other opinions, or please do bring up any other buildpack-vs-user process type concerns that we should capture here that I have missed.

Reply:
That makes sense to me!